readme/LinuxBuild.md

```diff
@@ -73,6 +73,8 @@ common available, therefore the Cairo PDF Export is without PDF Bookmarks.
 
 The binary executable will be placed in the `build/src/` subdirectory.
 
+After updating xournalpp, with perhaps `git pull`, execute `cmake ..` and `cmake --build .`.
```

Comment:
Acceptable?

Reply:
You should also mention to cd into the builddir. Also an "touch ../CMakeLists.txt" should be called before "cmake .."